tools/kraken.wdl

```diff
@@ -0,0 +1,339 @@
+## # Kraken2
+##
+## Methods for bootstrapping and running [Kraken2](https://github.com/DerrickWood/kraken2)
+
+version 1.0
+
+task download_taxonomy {
+    input {
+        Int memory_gb = 4
+        Int disk_size_gb = 60
+        Int max_retries = 3
+    }
+
+    parameter_meta {
+        memory_gb: "RAM to allocate for task"
+        disk_size_gb: "Disk space to allocate for task"
+        max_retries: "Number of times to retry in case of failure"
+    }
+
+    String db_name = "kraken2_taxonomy"
+
+    command <<<
+        set -euo pipefail
+
+        kraken2-build --download-taxonomy \
+            --use-ftp \
+            --db ~{db_name} 2>&1 \
+            | awk '/gunzip:/ { print; exit 42 } !/gunzip:/ { print }' 1>&2
+
+        tar -C ~{db_name}/ -czf "~{db_name}.tar.gz" .
+
+        rm -r ~{db_name}
+    >>>
+ 
+    runtime {
+        memory: memory_gb + " GB"
+        disk: disk_size_gb + " GB"
+        cpu: 1
+        docker: 'ghcr.io/stjudecloud/kraken2:branch-kraken2-1.0.0'
+        maxRetries: max_retries
+    }
+
+    output {
+        File taxonomy = db_name + ".tar.gz"
+    }
+
+    meta {
+        author: "Andrew Frantz"
+        email: "andrew.frantz@stjude.org"
+        description: "This WDL task downloads the Kraken2 taxonomy."
+    }
+}
+
+task download_library {
+    input {
+        String library_name
+        Int memory_gb = 4
+        Int added_disk_size_gb = 0
+        Int max_retries = 3
+    }
+
+    String db_name = "kraken2_"+library_name+"_library"
+
+    parameter_meta {
+        library_name: {
+            help: "Library to download. Note that due to the large size of the `nt` library, the `added_disk_size_gb` parameter might have to be used, depending on your backend. This option was not tested, but we estimate the size required to be around 2TB."
+            choices: ['archaea', 'bacteria', 'plasmid', 'viral', 'human', 'fungi', 'plant', 'protozoa', 'nt', 'UniVec', 'UniVec_Core']
+        }
```

Comment:
I think this is much cleaner to read. Assuming that it valid in the spec and well handled by various tools, then I think we should use this format.

Reply:
dxCompiler (https://github.com/dnanexus/dxCompiler) replaced it. `description` is an alias for `help` in dxCompiler, so I'd suggest we use that here.